lib/scss/inc/_aside.scss

```diff
@@ -114,35 +118,28 @@
   list-style: none;
   display: none;
 
+  .doc-link__active {
+    color: $doclink-active-color;
+  }
+
   & > li {
     & > a {
       display: block;
       transition: color 300ms;
-      padding: 5px 40px;
-      font-size: $font-size-base - 0.7;
-
-      &:not(.doc-link__active) {
-        color: $doclink-color;
-      }
+      color: $doclink-color;
+      padding: 5px 0;
+      font-size: $font-size-base - 2;
 
       &:hover {
         color: $doclink-hover-color;
-        background-color: $doclink-hover-bg-color
       }
     }
   }
 }
 
-.doc-link__active {
-  color: $doclink-active-color;
-  background-color: $doclink-active-bg-color;
-}
-
 .doc-link__header {
-  padding: 20px 40px;
+  padding: 20px 30px 20px 0;
   display: block;
-  font-weight: $font-weight-bold;
```

Comment:
Could we also keep `font-weight: $font-weight-bold;` and `text-transform: uppercase;` here as per Jayshree's feedback?